src/commands/create/templates.ts

```diff
@@ -2,7 +2,7 @@ import colors from "colors";
 
 export type Template = {
     name: string;
-    pkgManager?: "npm" | "yarn" | "pnpm";
+    pkgManager?: "npm" | "yarn" | "pnpm" | "go";
```

Comment:
Honestly, I'd rather remove this entry then. The `PackageManager` interface is very heavily opinionated towards the JavaScript environment. The whole "package manager" aspect of Go is only used when running `go mod download` to fetch all dependencies from the go.mod file.

There is no concept of registries/publish/link as all libraries can be pulled from git repositories.

What do you think about this?

Reply:
Also now I see (mistake on my part :( ) that it is an optional field. At first I added so I won't leave the `pkgManager` field empty. I am mostly sure that there is no need to add the `go` package manager

Reply:
I think that we will need to implement the publish/registries methods for go because we would like to publish the sdk packages to our own git registry. This will require changing the git config to use our tokens to log in to our git registry.

That being said, this is a future feature, which can be implemented later. You can leave this undefined for now.